docs_src/examples/Ch-ExamplesAddingMQTTDevice.md

```diff
@@ -28,520 +28,353 @@ This simulator has three behaviors:
+    };
+    publish( 'DataTopic', JSON.stringify(body));
+});
+
+// 2. Receive the reading request, then return the response
+// 3. Receive the put request, then change the device value
+subscribe( "CommandTopic" , (topic, val) => {
+    var data = val;
+        if (data.method == "set") {
+        message = data[data.cmd]
+    }else{
+        switch(data.cmd) {
+            case "ping":
+              data.ping = "pong";
+              break;
+            case "message":
+              data.message = message;
+              break;
+            case "randnum":
+                data.randnum = 12.123;
+                break;
+          }
     }
-
-    const deviceName = "MQ_DEVICE";
-    let message = "test-message";
-
-    // 1. Publish random number every 15 seconds
-    schedule('*/15 * * * * *', ()=>{
-        let body = {
-            "name": deviceName,
-               "cmd": "randnum",
-            "randnum": getRandomFloat(25,29).toFixed(1)
-        };
-        publish( 'DataTopic', JSON.stringify(body));
-    });
-
-    // 2. Receive the reading request, then return the response
-    // 3. Receive the put request, then change the device value
-    subscribe( "CommandTopic" , (topic, val) => {
-        var data = val;
-            if (data.method == "set") {
-            message = data[data.cmd]
-        }else{
-            switch(data.cmd) {
-                case "ping":
-                  data.ping = "pong";
-                  break;
-                case "message":
-                  data.message = message;
-                  break;
-                case "randnum":
-                    data.randnum = 12.123;
-                    break;
-              }
-        }
-        publish( "ResponseTopic", JSON.stringify(data));
-    });
+    publish( "ResponseTopic", JSON.stringify(data));
+});
+```
 
 To run the device simulator, enter the commands shown below with the
 following changes:
 
-1.  Replace the /path/to/mqtt-scripts in the example mv command with the
+- Replace the `/path/to/mqtt-scripts` in the example mv command with the
     correct path
+```
+$ mv mock-device.js /path/to/mqtt-scripts
+$ docker run -d --restart=always --name=mqtt-scripts \
+    -v /path/to/mqtt-scripts:/scripts  \
+    dersimn/mqtt-scripts --url mqtt://172.17.0.1 --dir /scripts
+```
+> The address `172.17.0.1` is point to the host of MQTT broker via the docker bridge network.
 
-2.  Replace the mqtt-broker-ip in the example docker run command with
-    the correct broker IP:
-
-        mv mock-device.js /path/to/mqtt-scripts
-        docker run -d --restart=always --name=mqtt-scripts \
-          -v /path/to/mqtt-scripts:/scripts  \
-          dersimn/mqtt-scripts --url mqtt://mqtt-broker-ip --dir /scripts
+## Prepare the Custom Configuration 
 
-### Setup
+In this section, we create a folder that contains files required for deployment:
+```
+- custom-config
+ |- mqtt.test.device.profile.yml
+ |- mqtt.test.device.config.toml
+```
```

Comment:
the profile yaml should be put to the `profiles` folder
the device toml should be put to the `devices` folder
https://github.com/edgexfoundry/device-mqtt-go/tree/v2.0.0/cmd/res

Reply:
Modified.